scripts/exregional_run_gsidiag.sh

```diff
@@ -189,8 +189,14 @@ else
   analworkname="_gsi"
 fi
```

Comment:
I see these in jobs/JREGIONAL_RUN_GSIDIAG. Why would you repeat these settings here?

Reply:
Should clean the repeated code in jobs/JREGIONAL_RUN_GSIDIAG.